### PR TITLE
makefile + made read_data.py read from validated instead of raw data

### PR DESCRIPTION
**Pull request by musiccabin**
@Psingh238 could you pls help test as usual? thx so much!

### CODE REVIEW

**Comment by Psingh238** [comment]
@musiccabin Were you unable to free up space to run Docker? I am running short on space as well myself now. Maybe instead of running Docker, you can create a conda environment to test everything as well? I will be trying to free up space today and try to test today. I will let you know how that goes.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+		--results_to ./results/tables/ \
+		--seed=522
+
+# render reports using Quarto
+$(REPORT_HTML): $(PLOTS) $(RESULTS) $(REFERENCES) ./reports/wine_quality_regressor_report.qmd
+	$(QUARTO) render ./reports/wine_quality_regressor_report.qmd --to html
+$(REPORT_PDF): $(PLOTS) $(RESULTS) $(REFERENCES) ./reports/wine_quality_regressor_report.qmd
+	$(QUARTO) render ./reports/wine_quality_regressor_report.qmd --to pdf
+
+all: $(DATA_RAW) $(DATA_CLEANED) $(TRAINING_SET) $(TEST_SET) $(RESULTS) $(PLOTS) $(REPORT_HTML) $(REPORT_PDF)
```
Comment (Psingh238) [inline Makefile:60]:
```suggestion
all: $(REPORT_HTML) $(REPORT_PDF)
```
Suggestion to simplify the Makefile `all` phony target. Moreover, convention is to have target `all` at the top before the first "real" target.

**Review by Psingh238 — COMMENTED** [review]
Suggested some changes for Makefile. Other changes look good too. I approve of the change to using the validated data CSV.

**Comment by musiccabin** [comment]
i quickly finished my other labs (though i'm pretty sure my grades will suffer slightly this block lol), so i can try to delete my conda course envs and see if i can run docker now. the local env has to have click installed and i don't have the exact same version of packages which i'm not sure if will cause issue?

**Pull request closed without merging**